### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by isaackumi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
-# atlas-base
\ No newline at end of file
+# atlas-base
+
+# atlas-base
+
+# User RESTful API
```
Comment (drewbrns) [inline README.md]: change this line to `RESTful API`, remove `User`

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+
+This api allows a user using a mobile client or create experiences as they live through the world.
```
Comment (drewbrns) [inline README.md]: remove this line

File: README.md [hunk 3]
```diff
@@ -1 +1,63 @@
-# atlas-base
\ No newline at end of file
+# atlas-base
+
+# atlas-base
```
Comment (drewbrns) [inline README.md]: Remove this line

**Comment by akornor** [comment]
Kindly squash multiple commits into one.

File: models/user.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        trim: true,
+        allowNull: false
+      },
+      isSuperUser:{
+        type: DataTypes.BOOLEAN,
+        defaultValue: false
+      },
+      role: {
+        type: DataTypes.ENUM,
+        values: ['las', 'admin', 'hostess', 'superuser']
```
Comment (drewbrns) [inline models/user.js]: remove `las`, `hostess` from the values

File: models/user.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+      },
+      lastName: DataTypes.STRING,
+      email: {
+        type: DataTypes.STRING,
+        unique: true,
+        allowNull: false
+      },
+      profile:{
```
Comment (drewbrns) [inline models/user.js]: remove `profile`

File: models/user.js [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+        type: DataTypes.STRING,
+        values: ['executiveBoss', 'ogaBoss', 'partyBoss', 'fashionBoss'],
+        allowNull: true
+      },
+      code: {
```
Comment (drewbrns) [inline models/user.js]: remove `code`

File: models/user.js [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+        type: DataTypes.STRING,
+        unique: true
+      },
+      password: DataTypes.STRING,
+      phoneNumber: {
+        type: DataTypes.STRING,
+        unique: true
+      },
+      language: DataTypes.STRING,
```
Comment (drewbrns) [inline models/user.js]: remove `language`, `area`, `age`, `smokerStatus` fields

File: controllers/user.js [hunk 8]
```diff
@@ -0,0 +1,310 @@
...
+        const {id} = req.params;
+        const admin = await models.User.getByPk(id)
+        admin.destroy({force: true})
+        res.status(410).end()
+    } catch (error) {
+        res.status(404).json({message: error.message})
+    }
+})
+
+router.post('/hostesses', auth, async (req, res) => {
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 9]
```diff
@@ -0,0 +1,310 @@
...
+        }
+        const { docs, pages, total } = await models.User.paginate(options)
+        const results = docs;
+        return res.status(200).json({results, pages, total})
+    } catch (error) {
+        return res.status(400).json({message: error.message})
+    }
+})
+
+router.get('/hostesses/:id', async (req, res) =>{
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 10]
```diff
@@ -0,0 +1,310 @@
...
+    try {
+        const {id} = req.params;
+        const hostess = await models.User.getByPk(id)
+        return res.status(200).json(hostess)
+    } catch (error) {
+        return res.status(400).json({message: error.message})
+    }
+})
+
+router.put('/hostesses/:id', async (req, res) => {
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 11]
```diff
@@ -0,0 +1,310 @@
...
+            return res.status(400).json({message: error.details[0].message})
+        }
+        const [rowId, [hostess]] =  await models.User.update(value, {returning: true, where: {id: req.params.id}})
+        res.status(200).json(hostess)
+    } catch (error) {
+        return res.status(400).json({message: error.message})
+    }
+})
+
+router.delete('/hostesses/:id', async (req, res) => {
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 12]
```diff
@@ -0,0 +1,310 @@
...
+            return res.status(404).json({message: "User not found."})
+        }
+        res.status(200).json(user)
+    } catch (error) {
+        return res.status(400).json({message: error.message})
+
+    }
+})
+
+router.post('/las', auth, async (req, res) => {
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpont

File: controllers/user.js [hunk 13]
```diff
@@ -0,0 +1,310 @@
...
+            return res.status(400).json({message: error.details[0].message})
+        }
+        const hostess =  await models.User.create({...value, role: 'las'})
+        return res.status(200).json(hostess) 
+    } catch (error) {
+        return res.status(404).json({message: error.message})
+    }
+})
+
+router.get('/las', auth, async (req, res) => {
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 14]
```diff
@@ -0,0 +1,310 @@
...
+        }
+        const { docs, pages, total } = await models.User.paginate(options)
+        const results = docs;
+        return res.status(200).json({results, pages, total})
+    } catch (error) {
+        return res.status(400).json({message: error.message})
+    }
+})
+
+router.get('/las/:id', auth, async (req, res) =>{
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 15]
```diff
@@ -0,0 +1,310 @@
...
+    try {
+        const {id} = req.params;
+        const las = await models.User.getByPk(id)
+        return res.status(200).json(las)
+    } catch (error) {
+        return res.status(400).json({message: error.message})
+    }
+})
+
+router.put('/las/:id', auth, async (req, res) => {
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 16]
```diff
@@ -0,0 +1,310 @@
...
+            return res.status(400).json({message: error.details[0].message})
+        }
+        const [rowId, [las]] =  await models.User.update(value, {returning: true, where: {id: req.params.id}})
+        res.status(200).json(las)
+    } catch (error) {
+        return res.status(400).json({message: error.message})
+    }
+})
+
+router.delete('/las/:id', async (req, res) => {
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 17]
```diff
@@ -0,0 +1,310 @@
...
+            return res.status(400).json({message: error.details[0].message})
+        }
+        const hostess =  await models.User.create({...value, role: 'hostess'})
+        return res.status(200).json(hostess) 
+    } catch (error) {
+        return res.status(404).json({message: error.message})
+    }
+})
+
+router.get('/hostesses', async (req, res) => {
```
Comment (drewbrns) [inline controllers/user.js]: remove this endpoint

File: controllers/user.js [hunk 18]
```diff
@@ -0,0 +1,310 @@
...
+
+router.post('/admins', async (req, res) => {
+    try {
+        const schema = {
+            email: Joi.string().email().required(),
+            firstName: Joi.string().required(),
+            lastName: Joi.string().required(),
+            phoneNumber: Joi.number().optional(),
+            password: Joi.string().required(),
+            code: Joi.string().optional()
```
Comment (drewbrns) [inline controllers/user.js]: remove `code`

File: controllers/user.js [hunk 19]
```diff
@@ -0,0 +1,310 @@
...
+            email: Joi.string().email().optional(),
+            firstName: Joi.string().optional(),
+            lastName: Joi.string().optional(),
+            phoneNumber: Joi.number().optional(),
+            smokerStatus: Joi.boolean().optional(),
+            language: Joi.string().optional(),
+            area: Joi.string().optional(),
+            isEnabled: Joi.boolean().optional(),
+            isVerified: Joi.boolean().optional(),
+            code: Joi.string().optional()
```
Comment (drewbrns) [inline controllers/user.js]: remove `code`, `smokerStatus`, `area`, `language`

**Comment by akornor** [comment]
Next time, I think it'll be good idea to have different additions in a separate PR.

**Pull request merged**